torch/_inductor/select_algorithm.py

```diff
@@ -27,6 +29,9 @@
 
 log = logging.getLogger(__name__)
 
+# cuda runtime does not work with "fork"
+multiprocessing.set_start_method("spawn", force=True)
```

Comment:
There are some warnings on `set_start_method` that it should be called from `__main__`, and can only be called once: https://docs.python.org/3/library/multiprocessing.html#multiprocessing.set_start_method

If I understand that right, we probably can't do this here?  But we probably can do `ctx = mp.get_context('spawn')` and then use `ctx.Process` etc.